k8s/prod/common/sscs/sscs-tribunals-api.yaml

```diff
@@ -30,11 +26,7 @@ spec:
         IDAM_URL: https://idam-api.platform.hmcts.net
         IDAM_OAUTH2_REDIRECT_URL: https://sscs-case-loader-prod.service.core-compute-prod.internal
         DOC_ASSEMBLY_URL: "http://dg-docassembly-prod.service.core-compute-prod.internal"
-        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
```

Comment:
this still needs to be here on tribunals until you upgrade your docker image version